internal/explorer/helpers.go

```diff
@@ -70,7 +72,7 @@ func (a *App) baseQuery(queryString string) (io.ReadCloser, error) {
 	if response.StatusCode != 200 {
 		var errResult interface{}
 		if err := json.NewDecoder(response.Body).Decode(errResult); err != nil {
-			return nil, fmt.Errorf("request failed: %w", errResult)
+			return nil, fmt.Errorf("request failed: %s", errResult)
 		}
 		return nil, fmt.Errorf("failed to query explorer network")
```

Comment:
Now here the decode of the return `Body` as error succeed, and now it's valid to return the `errResult` 
```suggestion
		return nil, fmt.Errorf("failed to query explorer network: %v", errResult)
```